kafka-admin/src/main/java/org/bf2/admin/kafka/admin/handlers/CommonHandler.java

```diff
@@ -76,6 +77,7 @@ public ErrorHandler(Class<T> causeType, Function<T, ResponseBuilder> mapper) {
             // 404 Not Found
             entry(UnknownTopicOrPartitionException.class, thrown -> errorResponse(thrown, ErrorType.TOPIC_NOT_FOUND)),
             entry(GroupIdNotFoundException.class, thrown -> errorResponse(thrown, ErrorType.GROUP_NOT_FOUND)),
+            entry(NotFoundException.class, thrown -> errorResponse(thrown, ErrorType.ERROR_NOT_FOUND)),
```

Comment:
Change to use the new error type
```suggestion
            entry(NotFoundException.class, thrown -> errorResponse(thrown, ErrorType.RESOURCE_NOT_FOUND)),
```

Reply:
Oh sorry, my mistake. Thanks for catching this Mike.